data.sql

```diff
@@ -8,3 +8,30 @@ values
   ('Boarmon', '2005-06-07', 20.4, true, 7),
   ('Blossom', '1998-10-13', 17, true, 3),
   ('Ditto', '2022-05-14', 22, true, 4);
+
+insert into owners(full_name, age)
+values
+  ('Sam Smith', 34),
+  ('Jennifer Orwell', 19),
+  ('Bob', 45),
+  ('Melody Pond', 77),
+  ('Dean Winchester', 14),
+  ('Jodie Whittaker', 38);
+
+insert into species(name)
+values
+  ('Pokemon'),
+  ('Digimon');
+
+update animals set species_id = case 
+  when name like '%mon' then (select id from species where name = 'Digimon')
+  else (select id from species where name = 'Pokemon')
+end;
+
+update animals set owner_id = case
+  when name = 'Agumon' then (select id from owners where full_name = 'Sam Smith')
+  when name = 'Pikachu' or name = 'Gabumon'  then (select id from owners where full_name = 'Jesnnifer Orwell')
+  when name = 'Plantmon' then (select id from owners where full_name = 'Bob')
+  when name = 'Charmander' or name = 'Squirtle' or name = 'Blossom' then (select id from owners where full_name = 'Melody Pond')
+  when name = 'Angemon' or name = 'Boarmon' then (select id from owners where full_name = 'Dean Winchester')
+  end;
```

Comment:
- [ ] according to the requirements you should add the below data to the tables.
- Insert the following data for vets:
    Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
    Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
    Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
    Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.
- Insert the following data for specialties:
    Vet William Tatcher is specialized in Pokemon.
    Vet Stephanie Mendez is specialized in Digimon and Pokemon.
    Vet Jack Harkness is specialized in Digimon.
- Insert the following data for visits:

    Agumon visited William Tatcher on May 24th, 2020.
    Agumon visited Stephanie Mendez on Jul 22th, 2020.
    Gabumon visited Jack Harkness on Feb 2nd, 2021.
    Pikachu visited Maisy Smith on Jan 5th, 2020.
    Pikachu visited Maisy Smith on Mar 8th, 2020.
    Pikachu visited Maisy Smith on May 14th, 2020.
    Devimon visited Stephanie Mendez on May 4th, 2021.
    Charmander visited Jack Harkness on Feb 24th, 2021.
    Plantmon visited Maisy Smith on Dec 21st, 2019.
    Plantmon visited William Tatcher on Aug 10th, 2020.
    Plantmon visited Maisy Smith on Apr 7th, 2021.
    Squirtle visited Stephanie Mendez on Sep 29th, 2019.
    Angemon visited Jack Harkness on Oct 3rd, 2020.
    Angemon visited Jack Harkness on Nov 4th, 2020.
    Boarmon visited Maisy Smith on Jan 24th, 2019.
    Boarmon visited Maisy Smith on May 15th, 2019.
    Boarmon visited Maisy Smith on Feb 27th, 2020.
    Boarmon visited Maisy Smith on Aug 3rd, 2020.
    Blossom visited Stephanie Mendez on May 24th, 2020.
    Blossom visited William Tatcher on Jan 11th, 2021.